serde_codegen/src/ser.rs

```diff
@@ -586,7 +525,6 @@ fn serialize_struct_variant(
         .with_fields(
             fields.iter().map(|field| {
                 builder.struct_field(field.ident.expect("struct has unnamed fields"))
-                    .with_attrs(field.attrs.iter().cloned())
```

Comment:
This line is weird. It was added [here](https://github.com/serde-rs/serde/commit/de89f95f3162a2f1404ed63940dc26ffa1bfbaf0#diff-3ffa97289153ae55d850a4b1effa6225R505). Let me know if you can figure out why. I didn't want to keep the `Vec<ast::Attribute>` just to use here. cc @erickt